robots/preservation_ingest/transfer_object.rb

```diff
@@ -80,8 +80,8 @@ def remove_bag(bag_pathname)
         # Note that symbolic links from /dor/export to /dor/workspace get
         #  translated into real files by use of --dereference
         def tarpipe_command(deposit_dir)
-          "ssh #{Dor::Config.transfer_object.from_host} " \
-            '"tar -C ' + "#{Dor::Config.transfer_object.from_dir} --dereference -cf - #{bare_druid}" + ' "' \
```

Comment:
here's fallout from the unintentional config change.

Reply:
From my perspective, this is intentional, so I need somebody to detail what the other intent is.

Reply:
s'ok. i fixed it with a commit.